useraccount.py

```
@@ -2,9 +2,11 @@
 import os
 import configparser
 
-project = '/data/project/himo'
+home_dir = os.getenv("HOME")
 # ---
-if not os.path.isdir(project):
+if home_dir and os.path.isdir(home_dir):
+    project = home_dir
+else:
     project = 'I:/core/bots/core1'
```

Ruff (0.8.2)

Use ternary operator `project = home_dir if home_dir and os.path.isdir(home_dir) else "I:/core/bots/core1"` instead of `if`-`else`-block

Replace `if`-`else`-block with `project = home_dir if home_dir and os.path.isdir(home_dir) else "I:/core/bots/core1"`

(SIM108)